torchmultimodal/models/clip/text_encoder.py

```diff
@@ -24,6 +24,7 @@ class CLIPTextEncoder(nn.Module):
         context_length (int): Maximum sequence length for Transforer.
         vocab_size (int): Vocab size.
         width (int): Embedding dimension for Transformer encoder.
+        dim_feedforward (int): Dimension of the feedfoward networks.
```

Comment:
did u check adding this isnt breaking anything internal in terms of args order

Reply:
I did a search on the codebase for CLIPTextEncoder and didn't find anywhere referencing the module that used positional arguments.